chore(server): (breaking) return an instance of `http.Server` from `createHTTPServer` in standalone adapter (#4014)
trpc/trpc: www/docs/server/adapter/standalone.md

## Going further

> `createHTTPServer` is returning an instance of Node's built-in `http.Server`(https://nodejs.org/api/http.html#class-httpserver), which means that you have an access to all it's properties and APIs. However, if `createHTTPServer` isn't enough for your usecase, you can also use the standalone adapter's `createHTTPHandler` function to create your own HTTP server. For instance:

Reviewer: ```suggestion
`createHTTPServer` returns an instance of Node.js's built-in [http.Server](https://nodejs.org/api/http.html#class-httpserver), giving you access to all it's properties and APIs. However, if `createHTTPServer` isn't enough for your use-case, you can also use the standalone adapter's `createHTTPHandler` function to create your own HTTP server. For instance:
```
